libraries/AP_NavEKF3/AP_NavEKF3.cpp

```diff
@@ -906,6 +958,45 @@ void NavEKF3::requestYawReset(void)
     }
 }
 
+/*
+  Update this instance error score value for all active cores
+*/
+float NavEKF3::updateCoreErrorScores()
+{
+    for (uint8_t i = 0; i < num_cores; i++) {
+        coreErrorScores[i] = core[i].errorScore();
+    }
+    return coreErrorScores[primary];
+}
+
+/*
+  Update the relative error for all alternate available cores with respect to primary core's error.
+  A positive relative error for a core means it has been more erroneous than the existing primary.
+  A negative relative error indicates a core which can be switched to.
+*/
+void NavEKF3::updateCoreRelativeErrors()
+{
+    float error = 0;
+    for (uint8_t i = 0; i < num_cores; i++) {
+        if(i != primary) {
```

Comment:
NitPick: should have a space after the "if"

Reply:
fixed, thanks